packages/webhooks/src/signature-validator.ts

```
@@ -0,0 +1,228 @@
+    const hmac = createHmac(algorithm, secret);
+    hmac.update(payload);
+    return hmac.digest('hex');
+  }
+
+  /**
+   * Generate signature for outgoing webhooks (for testing)
+   */
+  generateSignature(
+    payload: string | Buffer, 
+    secret: string, 
+    algorithm: string = 'sha256'
```

ESLint

[error] Type string trivially inferred from a string literal, remove type annotation.

(@typescript-eslint/no-inferrable-types)